CHANGELOG.md

```diff
@@ -4,6 +4,18 @@
 
 * PayPal
     * Add optional property `PayPalCheckoutRequest.setUserAuthenticationEmail()`
+* ShopperInsights
+  * Requires opt in - `@OptIn(ExperimentalBetaApi::class)`
+  * Add `ShopperInsightsClient.getRecommendedPaymentMethods()` for returning recommendations based on the buyer
+  * Add `ShopperInsightsClient.sendPayPalPresentedEvent()`
+  * Add `ShopperInsightsClient.sendPayPalSelectedEvent()`
+  * Add `ShopperInsightsClient.sendVenmoPresentedEvent()`
+  * Add `ShopperInsightsClient.sendVenmoSelectedEvent()`
+  * Add `ShopperInsightsRequest`
+  * Add `ShopperInsightsBuyerPhone`
+  * Add `ShopperInsightsCallback`
+  * Add `ShopperInsightsResult`
+  * Add `ShopperInsightsInfo`
```

Comment:
```suggestion
* Add ShopperInsights module (BETA)
  * Requires opt in - `@OptIn(ExperimentalBetaApi::class)`
```

This aligns more with how iOS is doing the CHANGELOG - and since the entire module is new I don't think we necessarily need to call out every new class/method. What do you think?

Reply:
Yep, that makes sense. I agree with omitting the individual classes.

I was looking at when `PayPalNativeCheckout (BETA)` was added, it seemed had a bullet for everything.